### DISCUSSION TITLE
Need-coKreat release cycle dates post 6.0.0

### DISCUSSION THREAD

**Discussion by omkar309** [Sunbird-coKreat/Community#89] [General Discussions]
Hey @mmanimegalai 
Can we get the [release calendar](https://docs.google.com/spreadsheets/d/1bw0YMXBATpaL0u8iDc6uE4kIxKtkvrABxc0ruud5-7o/edit#gid=0) updated with dates for next release cycles post 6.0.0? This will help us plan developments in the month of July '23.

**Comment by mmanimegalai** [comment]
@omkar309 
Yes, we have the dates for the next major and the intermediate cycles. will have the release calendar updated in the microsite by eod

**Reply by omkar309** [reply]
Thanks!

**Comment by mmanimegalai** [comment]
@omkar309 The release calendar has been updated in the microsite. Please note that there are some nuances that are being sorted out, will keep you posted. However you can go ahead with your planning based on these dates
cc @vinukumar-vs

**Reply by omkar309** [reply]
Understood. Thanks for the update @mmanimegalai

**Comment by mmanimegalai** [comment]
@omkar309 @abhay-samagra

The release dates for 6.1.0 & beyond has been revised. Please find the updates in the coKreat micrositer
Also there is a delay in the coKreat 6.0.0 Regression which may have an impact on the 6.0.0 release. Please note that SB ED regression is also getting delayed that might have an impact on coKreat 6.0.0 release 

cc @vinukumar-vs

**Reply by omkar309** [reply]
Thanks for the update @mmanimegalai .
